lib/custom/src/MShop/Service/Provider/Payment/OmniPay.php

```diff
@@ -711,11 +737,17 @@ protected function getData( \Aimeos\MShop\Order\Item\Base\Iface $base, $orderid,
 	 */
 	protected function getProvider()
 	{
+		$config = $this->getServiceItem()->getConfig();
+
+		if($this->getValue( 'apiKey', false ) != false){
+			$config['apiKey'] = $this->getValue( 'apiKey', false );
```

Comment:
Why do you use an if-condition here? The value of "apiKey" would be passed to initialize() nevertheless if available as long as it has no "stripe." prefix (which is OK in config settings).

Reply:
Because of I added it to parent OmniPay class. So we add apiKey to an array only if child payment provider need it. Okay I can add getProvider() to child Stripe class and do it there, without if-condition